alsi/mappings.py

```diff
@@ -0,0 +1,102 @@
+from enum import Enum
+from typing import Union
+from .exceptions import InvalidCountryException
+
+
+def retrieve_country(input: Union["Area", str]) -> "Area":
+
```

Comment:
Please remove empty line.

Reply:
Line number 7?

Reply:
Yes, GitHub comments appear under the commented line apparently.